app/utils/UnloadingFindingsAnswersHelper.scala

```diff
@@ -130,6 +133,17 @@ class UnloadingFindingsAnswersHelper(userAnswers: UserAnswers, referenceDataServ
     call = Some(controllers.routes.SessionExpiredController.onPageLoad())
   )
 
+  def additionalReference(index: Index): Option[SummaryListRow] = getAnswerAndBuildRow[AdditionalReference](
+    page = AdditionalReferencePage(index),
+    formatAnswer = formatAsText,
+    prefix = "unloadingFindings.additional.reference",
+    args = index.display,
+    id = Some(s"change-additional-reference-${index.display}"),
+    call = Some(controllers.routes.UnloadingTypeController.onPageLoad(arrivalId, CheckMode)) //TODO change me please
```

Comment:
Can just leave this as `None` for now. The change links don't need to be functional

Reply:
Yeah, we added a Some(blah) because the AC mentioned a non-functional CL